src/store/plugins/sdk.js

```diff
@@ -174,19 +173,14 @@ export default (store) => {
         });
         commit('setSdkReady');
       },
+      async changeNode(_, network) {
+        await watchUntilTruthy(() => store.getters['sdkPlugin/sdk']);
+        sdk.pool.delete(network.name);
+        sdk.addNode(network.name, await Node({ url: network.url }), true);
+      },
```

Comment:
Declared this method but called only once. Doesn't need tobe a function. Code could be inline.